twake/backend/node/src/core/platform/services/database/services/orm/connectors/cassandra/typeTransforms.ts

```diff
@@ -38,6 +38,10 @@ export const transformValueToDbString = (
     return `${parseInt(v)}`;
   }
   if (type === "uuid" || type === "timeuuid") {
+    if (isNull(v)) {
+      return "null";
+    }
+
```

Comment:
Why is it needed?
If you do this kind of modification you must at least also do it in mongo connector.

Also, are we able to put "NULL" in a scylladb timeuuid / uuid column ?

Reply:
Already done too 👍🏻